test/zone_test.rb

```diff
@@ -48,6 +48,9 @@ def test_zone_has_config_with_provider
   end
 
   def test_zone_is_invalid_if_provider_does_not_support_record_type
+    supported_record_types = Provider::DynECT.record_types - ['ALIAS']
+    Provider::DynECT.stubs(:record_types).returns(supported_record_types)
```

Comment:
it doesn't actually HAVE to be an apex... you CAN define an ALIAS that is NOT an apex